RFC-010-core-strategy/RFC-010-core-strategy.md

```diff
@@ -0,0 +1,126 @@
+# RFC010 - Core Strategy
+
+## Proposed by
+
+Felicity Brand, Carrie Crowe
+
+## Current status
+
+See the status indicated on the pull request description.
+
+
+## Proposal overview
+
+This RFC is to gain consensus on several key areas of The Good Docs Project as it stands in 2022. This touches on our vision, audience, content and how we deliver value. 
+The information is contained in the Core Strategy document (core-strategy.md).
+
+The Core Strategy document seeks to outline the fundamentals of the project, answering the following questions:
+
+* What is The Good Docs Project?
+* What do we do?
+* Who is our audience?
+* What content do we offer?
+
+This can then drive our content strategy, and work will follow in future RFCs to answer deeper questions about content substance, workflow, structure and governance.
+
+
+## Motivation
+
+There are several benefits to articulating where we are, how we can best support our users and community members, and where we’d like to grow in future.
+
+Documenting our core strategy means that:
+
+**As a project**, we can: 
+
+* Have a touchstone or “North Star” to guide decisions about content produced by the project, as well as communications about the project.
+* Reinforce the foundation that is already underlying our current initiatives.
+* Adapt easily to support our audience. As we grow, our audience will grow and their needs might change. 
+
+**As a community**, we can:
+
+* Hold a shared idea of what we’re about—so that we’re on the same page. 
+* Present a unified idea, and use consistent language, when talking about the identity of our project to others.
+
+**As content producers**, we can: 
+
+* Address content challenges, prioritize them accordingly and form solutions. 
+* Tie each piece of content that we create back to a community or project goal. 
+
+
+## Proposal
+
+That we accept the following related file:
+
+* core-strategy.md
+
+
+## Consequences
+
+An agreed core strategy will help all members of our e annually, or if we happen to have a major shift in focus or experience unexpected growth.

```

Comment:
```suggestion
An agreed core strategy will help all members of our [e annually], or if we happen to have a major shift in focus or experience unexpected growth.

```
Seems like some bits of the sentence (see bracket) went missing in the editing process. Can you check this sentence again?

Reply:
Copy/paste error. So embarrassing 😊 Will fix!